santoku/aws/sqs_handler.py

```diff
@@ -1,7 +1,6 @@
-import boto3
-
 from typing import Any, Dict, List
 
+import boto3
```

Comment:
As a good practice we made the convention to add full imports at the top and then partial imports as they override the namespace. The order should then be:

import <standard library>

import <third party library>

from <std lib> import something

from <3rd party lib> import something